src/blockchain/extrinsic.ts

```diff
@@ -92,7 +92,7 @@ export class Extrinsic<T extends ISubmittableResult = ISubmittableResult, C exte
   public payWithCapacity(nonce?: number): Promise<ParsedEventResult> {
     return firstValueFrom(
       this.api.tx.frequencyTxPayment
-        .payWithCapacity(this.extrinsicCall)
+        .payWithCapacity(this.extrinsicCall.toHex())
```

Comment:
I think SubmittableExtrinsic will need a check as this is no longer accepted 

Note: I will test if it works.

Reply:
This might not work if our prior test data is based on pre-runtime v110